bandit/plugins/injection_sql.py

```diff
@@ -92,8 +92,25 @@ def _evaluate_ast(node):
     elif hasattr(ast, "JoinedStr") and isinstance(
         node._bandit_parent, ast.JoinedStr
     ):
-        statement = node.s
-        wrapper = node._bandit_parent._bandit_parent
+        if hasattr(ast, 'unparse'):
```

Comment:
pyflakes seems to be able to handle `JoinedStr` https://github.com/PyCQA/pyflakes/blob/e19886e583637a7e2eec428cc036094b9630f2d0/pyflakes/checker.py#L1740-L1753 why can't we do something similar here without `unparse`?

Reply:
Thanks for looking at my pull request!
Do you mean something like this? Now it works for all versions of python.